### ISSUE TITLE
"Not clickable" error message is not JSON when ErrorHandler.check_response expects JSON

### ISSUE THREAD

**Issue by vr2262** [SeleniumHQ/selenium#1470]
I saw an error like this one:

```
  File "/home/travis/virtualenv/python3.5.0/lib/python3.5/site-packages/selenium/webdriver/remote/webelement.py", line 75, in click

    self._execute(Command.CLICK_ELEMENT)

  File "/home/travis/virtualenv/python3.5.0/lib/python3.5/site-packages/selenium/webdriver/remote/webelement.py", line 454, in _execute

    return self._parent.execute(command, params)

  File "/home/travis/virtualenv/python3.5.0/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 201, in execute

    self.error_handler.check_response(response)

  File "/home/travis/virtualenv/python3.5.0/lib/python3.5/site-packages/selenium/webdriver/remote/errorhandler.py", line 102, in check_response

    value = json.loads(value_json)

  File "/opt/python/3.5.0/lib/python3.5/json/__init__.py", line 319, in loads

    return _default_decoder.decode(s)

  File "/opt/python/3.5.0/lib/python3.5/json/decoder.py", line 339, in decode

    obj, end = self.raw_decode(s, idx=_w(s, 0).end())

  File "/opt/python/3.5.0/lib/python3.5/json/decoder.py", line 357, in raw_decode

    raise JSONDecodeError("Expecting value", s, err.value) from None

json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

I did some digging and it looks like [this line](https://github.com/SeleniumHQ/selenium/blob/30a541659e3e72c64f159f3da8cb06ff28e5a362/py/selenium/webdriver/remote/errorhandler.py#L102) is choking because the value of `value_json` was:

```
'Element is not clickable at point (156, 457.79998779296875). Other element would receive the click: <div class="..."></div>'
```

**Comment by roddux** [comment]
I have just encountered this issue, too.

Context: Selenium using Firefox in an attempt to click on a marker on Google maps. Apparently the `.marker` selector does not give a clickable element!

**Comment by lukeis** [comment]
this was fixed with 264131521ed2392f47d3b661155ac97107c41f63  pending another release of the python library.

The other issue with the element not being clickable is still persistent and tracked in issue #1202
